stdlib/public/core/StringUTF8.swift

```diff
@@ -725,3 +715,14 @@ extension String.UTF8View : CustomPlaygroundQuickLookable {
   }
 }
 
+extension String {
+  @available(*, unavailable, message: "Please use String.utf8CString instead.")
+  public var nulTerminatedUTF8: ContiguousArray<UTF8.CodeUnit> {
+    Builtin.unreachable()
+  }
+
+  @available(*, unavailable, renamed: "utf8CString")
+  public var nulTerminatedUTF8CString: ContiguousArray<CChar> {
+    Builtin.unreachable()
+  }
+}
```

Comment:
You don't need to mark nulTerminatedUTF8CString unavailable, just delete it. It wasn't on tree long enough for anyone to use it.